gym/spec/build_command_generator_spec.rb

```diff
@@ -22,7 +22,7 @@
       xcargs = xcargs_hash.map do |k, v|
         "#{k.to_s.shellescape}=#{v.shellescape}"
       end.join ' '
-      options = { project: "./examples/standard/Example.xcodeproj", sdk: "9.0", xcargs: xcargs, scheme: 'Example' }
+      options = { project: "./examples/standard/Example.xcodeproj", sdk: "9.0", toolchain: "com.apple.dt.toolchain.Swift_2_3", xcargs: xcargs, scheme: 'Example' }
```

Comment:
Examples about toolchains.
It can be also org.swift.xxxx.

| With Xcode 8 Toolchain | env |
| --- | --- |
| <img width="360" src="https://cloud.githubusercontent.com/assets/932290/18035150/52226884-6d8a-11e6-90a2-ab7d46940ec0.png"> | ![screen shot 2016-08-29 at 1 43 53 am](https://cloud.githubusercontent.com/assets/932290/18035151/54a60aa2-6d8a-11e6-978d-37e79125e5ff.png) |

| With Swift 3.0 Preview 6 Snapshot | env |
| --- | --- |
| <img width="360" src="https://cloud.githubusercontent.com/assets/932290/18035171/c141d07e-6d8a-11e6-8928-3fa5af10561c.png"> | ![screen shot 2016-08-29 at 1 43 08 am](https://cloud.githubusercontent.com/assets/932290/18035172/c253759e-6d8a-11e6-9d99-961176968034.png) |